misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/AnyExpectationsSpec.kt

```diff
@@ -599,7 +599,7 @@ abstract class AnyExpectationsSpec(
 
 
     describeFun(notToEqualNullFeature, notToEqualNull) {
-        val notToEqualNullFunctions = unifySignatures(notToEqualNullFeature, notToEqualNull)
+        val notToEqualNullFunctions = unifySignatures<Int?, Int>(notToEqualNullFeature, notToEqualNull)
```

Comment:
I'm not sure why this does not work without explicit type parameters, however, `unifySignatures` looks more like a corner-case rather than real-life usage.